toolkit/scss/search/_search-summary.scss

```diff
@@ -28,4 +28,8 @@
   @include media(tablet) {
     @include core-16;
   }
-}
\ No newline at end of file
+}
+
+.search-summary-accessible-hint-wrapper {
```

Comment:
My only complaint is that the word 'wrapper' means pretty much nothing. If most-things-like-this-thing get the word "wrapper" on the end, then it's fine I guess.

Reply:
It's because it's the level *up* from the `.search-summary-accessible-hint`